OWNERS

```diff
@@ -18,6 +18,5 @@ approvers:
   - knobunc
   - trozet
 
-# Bugzilla info; "openshift-sdn" is wrong but there is no "generic networking" subcomponent
 component: Networking
-subcomponent: openshift-sdn
+subcomponent: cluster-network-operator
```

Comment:
does this take effect in JIRA same way it used to in bugzilla?
If so don't we need to do Networking/cluster-network-operator or something? or are we sure this works..